RamilyaNigmatullina/pdp-2
# [#159718578] Add pg_search gem

# Summary

Add PG Search gem. 
Add complex search by several related models for several fields. You can search hotels by name, address, city and country. 
https://www.pivotaltracker.com/story/show/159718578

# Test plan

List of steps to manually test introduced functionality:

* Go to Application
* Fill in Search field with hotel name or hotel address
* Click on Find link
* See the search results match search query
* Fill in Search field with city or country where hotel is located in
* See the search results match search query
